test/tokens/BurnableXcert.test.js

```diff
@@ -8,29 +9,31 @@ contract('BurnableXcert', (accounts) => {
   let id2 = web3.sha3('test2');
   let id3 = web3.sha3('test3');
   let id4 = web3.sha3('test4');
-  let mockProof = "1e205550c271490347e5e2393a02e94d284bbe9903f023ba098355b8d75974c8";
+  const proof = '1e205550c271490347e5e2393a02e94d284bbe9903f023ba098355b8d75974c8';
+  let config = [web3Util.padLeft(web3Util.numberToHex(1821195657), 64)];
```

Comment:
How did you get the 182119565?

Reply:
Ah, ok it's a timestamp

Reply:
Use `cost` instead of `let` when possible.